src/python/pants/jvm/resources.py

```diff
@@ -89,9 +91,21 @@ async def assemble_resources_jar(
         ProcessResult,
         Process(
             argv=[
-                zip.path,
-                output_filename,
-                *sorted(input_files),
+                bash.path,
+                "-c",
+                " ".join(
```

Comment:
It may be worth using `shlex.join` here, just in case any of `input_files` have dubious filenames.

Reply:
done